target/linux/ar71xx/image/tp-link.mk

```diff
@@ -426,7 +426,7 @@ TARGET_DEVICES += tl-wpa8630
 define Device/tl-wr703n-v1
     $(Device/tplink-4mlzma)
     DEVICE_TITLE := TP-LINK TL-WR703N
-    DEVICE_PACKAGES := kmod-usb-core kmod-usb2
+    DEVICE_PACKAGES := kmod-usb-core kmod-usb-chipidea
     BOARDNAME := TL-WR703N
```

Comment:
please do not change the default package selection here. people will expect host mode as the default. happy to merge this once this bit has been fixed.

@neykov does usb gadget support dual role devices ? i would like to have a 703 i can use for mobile debugging, but that would require the usb device to expose an ethernet and console endpoint.